### PR TITLE
`use-deliver`: Review practical sessions

### PR DESCRIPTION
**Pull request by molecula2788**

### CODE REVIEW

File: chapters/use-deliver/practical-sessions/reading/README.md [hunk 1]
```diff
@@ -42,7 +39,7 @@ Ideally, each practical activity (quiz, tutorial, drill) would be automatically
 For example, in Computer Programming courses, drills and tutorials typically require the learner to write programs.
 Checkers should accompany these activities to ensure automated verification.
 Quizzes, if employed, are automatically checked for correctness.
-Therefore, for humanitarian fields of study such as history or philosophy, we recommend using quizzes instead of drills as a form of practical activity.
+Therefore, for fields of study pertaining to the humanities, such as history or philosophy, we recommend using quizzes instead of drills as a form of practical activity.
```
Comment (teodutu) [inline chapters/use-deliver/practical-sessions/reading/README.md:42]: All this paragraph needs to be reworded and expanded. Now we see drills as being all types of work that students may do and we split them between questions (previously named quizzes) and tasks (larger, more in-depth, require full solutions). You can see the differences highlighted in my PR: https://github.com/open-education-hub/methodology/pull/29

**Review by teodutu — CHANGES_REQUESTED** [review]
You should incorporate the changes to drills and quizzes.

**Review by RazvanN7 — CHANGES_REQUESTED** [review]
what @teodutu said.

**Review by teodutu — APPROVED** [review]
Yeah I think it's clear enough now

**Review by RazvanN7 — APPROVED** [review]

**Pull request merged**